python/lsst/ip/isr/overscan.py

```diff
@@ -974,3 +873,473 @@ def debugView(self, image, model, amp=None, isTransposed=True):
+                Mean of the fit serial overscan region.
+                This and the following values will be tuples of
+                (serial, parallel) if doParallelOverscan=True.
+            ``overscanMedian``
+                Median of the fit serial overscan region.
+            ``overscanSigma``
+                Sigma of the fit serial overscan region.
+            ``residualMean``
+                Mean of the residual of the serial overscan region after
+                correction.
+            ``residualMedian``
+                Median of the residual of the serial overscan region after
+                correction.
+            ``residualSigma``
+                Mean of the residual of the serial overscan region after
+                correction.
+
+
+        Raises
+        ------
+        RuntimeError
+            Raised if an invalid overscan type is set.
+        """
+        # Do Serial overscan first.
+        serialOverscanBBox = amp.getRawSerialOverscanBBox()
+        imageBBox = amp.getRawDataBBox()
+
+        if self.config.doParallelOverscan:
+            # We need to extend the serial overscan BBox to the full
+            # size of the detector.
+            parallelOverscanBBox = amp.getRawParallelOverscanBBox()
+            imageBBox = imageBBox.expandedTo(parallelOverscanBBox)
+
+            if isTransposed:
+                serialOverscanBBox = geom.Box2I(
+                    geom.Point2I(serialOverscanBBox.getMinX(), imageBBox.getEndY()),
+                    geom.Extent2I(imageBBox.getWidth(), serialOverscanBBox.getHeight()),
+                )
+            else:
+                serialOverscanBBox = geom.Box2I(
+                    geom.Point2I(serialOverscanBBox.getMinX(),
+                                 imageBBox.getMinY()),
+                    geom.Extent2I(serialOverscanBBox.getWidth(),
+                                  imageBBox.getHeight()),
+                )
+
+        serialResults = self.correctOverscan(
+            exposure,
+            amp,
+            imageBBox,
+            serialOverscanBBox,
+            isTransposed=isTransposed,
+            leadingToSkip=self.config.leadingColumnsToSkip,
+            trailingToSkip=self.config.trailingColumnsToSkip,
+        )
+        overscanMean = serialResults.overscanMean
+        overscanMedian = serialResults.overscanMedian
+        overscanSigma = serialResults.overscanSigma
+        residualMean = serialResults.overscanMeanResidual
+        residualMedian = serialResults.overscanMedianResidual
+        residualSigma = serialResults.overscanSigmaResidual
+
+        # Do Parallel Overscan
+        parallelResults = None
+        if self.config.doParallelOverscan:
+            # This does not need any extensions, as we'll only
+            # subtract it from the data region.
+            parallelOverscanBBox = amp.getRawParallelOverscanBBox()
+            imageBBox = amp.getRawDataBBox()
+
+            maskIm = exposure.getMaskedImage()
```

Comment:
I think this `maskIm` can be dropped here and below.  It's almost certainly a leftover from a previous attempt at fixing this.